src/styles/table/table.css

```diff
@@ -0,0 +1,40 @@
+@import url(../../themes/themes.css);
+/*
+ *  Styling for native-HTML tables.
+ **/
+
+table {
+  border-collapse: collapse;
+}
+
+tr {
+  background-color: var(--surface);
+  border-bottom: 1px solid var(--surface-2dp);
+}
+
+tr:hover {
+  background-color: var(--surface-1dp);
+}
+
+tr:hover tr:hover {
+  background-color: var(--surface-2dp);
+}
+
+td,th {
+  color: var(--on-surface);
+  padding: 10px;
+}
+
+td {
+  font-size: 0.813em;
+  vertical-align: middle;
+}
+
+th {
+  font-size: 0.75em;
+  opacity: 60%;
```

Comment:
Why set opacity? If you want it to stand out maybe set color: var(--primary)?

Reply:
From https://material.io/design/color/dark-theme.html#ui-application:

When light text appears on dark backgrounds (shown here as white on black) it should use the following opacity levels:

High-emphasis text has an opacity of 87%
Medium-emphasis text and hint text have opacities of 60%
Disabled text has an opacity of 38%